rpc/region_request.go

```diff
@@ -141,7 +141,11 @@ func (s *RegionRequestSender) onSendFail(bo *retry.Backoffer, ctx *locate.RPCCon
 	if errors.Cause(err) == context.Canceled {
 		return errors.Trace(err)
 	}
-	if grpc.Code(errors.Cause(err)) == codes.Canceled {
+	code := codes.Unknown
+	if s, ok := status.FromError(errors.Cause(err)); ok {
```

Comment:
why old way can't work?

Reply:
It still can work, but it has already been deprecated. See https://github.com/grpc/grpc-go/blob/master/rpc_util.go#L688.
```
// Deprecated: use status.Code instead.
```